website/src/views/covid.ts

```diff
@@ -294,6 +296,10 @@ export class CovidSequencingEffortsView extends BaseView<
                         location: {},
                         dateRange: constants.defaultDateRange,
                     },
+                    variantFilter: {
+                        lineages: {},
+                        mutations: {},
```

Comment:
But it's probably ok for now - at least the implementation is less effort than if we would introduce another filter type.

Reply:
I'm not sure I fully understand you - do you mean that the filter is labeled "Variant filter", or is this visible somewhere else that I am missing? Otherwise, I agree - it would be great to be able to customize the label of the Variant Filter for each page.

Reply:
I mean: The type has `mutations`, but they will never be used on sequencing efforts pages, because we don't have a mutations filter there.